Google.PowerShell/Google.PowerShell.csproj

```diff
@@ -195,9 +195,9 @@
       <SubType>Designer</SubType>
     </None>
     <None Include="README.md" />
-    <None Include="ReleaseFiles\BootstrapGoogleCloudPowerShell.ps1" />
+    <None Include="ReleaseFiles\BootstrapCloudToolsForPowerShell.ps1" />
     <None Include="ReleaseFiles\GoogleCloudPlatform.Format.ps1xml" />
-    <None Include="ReleaseFiles\GoogleCloudPowerShell.psd1" />
+    <None Include="ReleaseFiles\CloudToolsForPowerShell.psd1" />
```

Comment:
Good catch. Evidently you need to "Save All" to save changes to the project file. (Which is something I'm sure I've been burned by in the past.)